src/PrestaShopBundle/Bridge/AdminController/Action/HeaderToolbarAbstractAction.php

```diff
@@ -29,8 +29,8 @@
 namespace PrestaShopBundle\Bridge\AdminController\Action;
 
 /**
- * This class is the object to instantiate if you want to add a bulk action for your list.
+ * Represents an action in the header toolbar of the page
  */
-class ListBulkAction extends Action
+class HeaderToolbarAbstractAction extends AbstractAction
```

Comment:
```suggestion
class HeaderToolbarAction extends AbstractAction
```

Reply:
This one is not abstract, probably PHPStorm who renamed it when you renamed the `AbstractAction`